xcodebuild.go

```diff
@@ -283,21 +283,21 @@ func handleTestRunError(prevRunParams testRunParams, prevRunResult testRunResult
 		if isStringFoundInOutput(errorPattern, prevRunResult.xcodebuildLog) {
 			log.Warnf("Automatic retry reason found in log: %s", errorPattern)
 			if prevRunParams.retryOnTestRunnerError {
-				log.Printf("retryOnTestRunnerError=true - retrying...")
+				log.Printf("Automatic retry is enabled - retrying...")
 
 				prevRunParams.buildTestParams.RetryTestsOnFailure = false
 				prevRunParams.retryOnTestRunnerError = false
 				return cleanOutputDirAndRerunTest(prevRunParams)
 			}
 
-			log.Errorf("retryOnTestRunnerError=false, no more retry, stopping the test!")
+			log.Errorf("Automatic retry is disabled, no more retry, stopping the test!")
 			return prevRunResult.xcodebuildLog, prevRunResult.exitCode, prevRunResult.err
 		}
 	}
 
-	if prevRunParams.xcodeMajorVersion < 13 && prevRunParams.buildTestParams.RetryTestsOnFailure {
```

Comment:
The `prevRunParams.xcodeMajorVersion < 13` check is no longer required as it's already performed in `ProcessConfig()`. By the time this line is executed, the check should have already been performed.